changes/20240828133012.feature

```diff
@@ -0,0 +1 @@
+✨ Added LsRecursive
```

Comment:
expand on this, explain what it is for (and how it is different from the existing functions) but keep it one line